modules/acquia_cms_common/config/optional/filter.format.filtered_html.yml

```diff
@@ -22,14 +22,14 @@ filters:
     id: filter_url
     provider: filterclear
     status: true
-    weight: -47
+    weight: -46
```

Comment:
@panshulK Shouldn't we revert this file changes completely unless weight change is required ?
cc: @chandan-singh7929

Reply:
The weight changes are required.
Because the preview is appearing only when the selected filters are re-ordered.